src/Microsoft.AspNetCore.SignalR.Common/Protocol/HandshakeResponseMessage.cs

```diff
@@ -11,21 +11,36 @@ public class HandshakeResponseMessage : HubMessage
         /// <summary>
         /// An empty response message with no error.
         /// </summary>
-        public static readonly HandshakeResponseMessage Empty = new HandshakeResponseMessage(null);
+        public static readonly HandshakeResponseMessage Empty = new HandshakeResponseMessage(error: null);
 
         /// <summary>
         /// Gets the optional error message.
         /// </summary>
         public string Error { get; }
 
+        /// <summary>
+        /// Highest minor protocol version that the server supports.
+        /// </summary>
+        public int MinorVersion { get; }
+
         /// <summary>
         /// Initializes a new instance of the <see cref="HandshakeResponseMessage"/> class.
+        /// If you are intending to send back an error, the minor version doesn't matter.
```

Comment:
Suggested rephrase: An error response does not include a minor version, since the handshake has failed.